### ISSUE TITLE
question about using markdown for asset description

### ISSUE THREAD

**Issue by OlgaKrebs** [seek4science/seek#895]
I woud like to add image to asset description, but how to do this?  it does expect URL : " ![](http://)"  Can I just upload image to .... any place and use it

**Comment by fbacall** [comment]
the syntax is:
`[Description of the image](https://url-to-the-image.com/img.png)`

the image can be anywhere on the web, just keep in mind that it is not cached anywhere in SEEK and so could disappear in the future

**Comment by OlgaKrebs** [comment]
so user HAS TO put an image on the web first? image on local mashine is not usable?  this is request from user

**Comment by fbacall** [comment]
yes, there is no way to upload an image (unless the user uploads it as a data file or document etc.)

**Comment by fbacall** [comment]
> I TRIED both : upload image as a doc or link to image file in NextCloud - result here https://fairdomhub.org/investigations/566 - no one is shown

Can you make this public?

**Comment by fbacall** [comment]
OK, I wasn't logged in.

It seems like neither the URLs in the markdown point to images.

Not sure what the correct link for the NextCloud file would be, but for the Document on SEEK you need use the URL to the actual image itself.

You can get this by clicking "View Content" and then right click the image that appears and click (depending on your browser) "Copy Image Link". That document is not visible to me so I cannot get the exact link, but it would look something like:
`https://fairdomhub.org/documents/4064/content_blobs/1234/download?disposition=inline&image_size=600`

**Comment by OlgaKrebs** [comment]
thanks, Finn - for SEEK image it works now
